torchci/components/WorkflowBox.tsx

```diff
@@ -174,7 +183,22 @@ export default function WorkflowBox({
             ></input>
             <input type="submit" value="Search"></input>
           </form>
+          <div
+            style={{
+              // Ensures elements after this div are actually below it (due to float)
+              clear: "both",
+            }}
+          ></div>
           {searchString && <div>{searchRes.info}</div>}
+          <div style={{ margin: ".5em 0em" }}>
```

Comment:
Consider putting the tests above the "search raw logs" box. Otherwise it seems like the search box is for the tests, especially once they're expanded

<img width="2066" alt="image" src="https://github.com/pytorch/test-infra/assets/4468967/31ed8bbb-6e8e-46a2-a1f6-fbde28ffeb78">